### PR TITLE
Create test.js

### PR DESCRIPTION
**Pull request by VipinDevelops**

### CODE REVIEW

**Comment by VipinDevelops** [comment]
hello please review this pR

**Comment by Twitter-Dev** [comment]
`this is code` it supports markdown!

**Comment by Twitter-Dev** [comment]
Hey From Slack!

File: test.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("hello world")
```
Comment (Twitter-Dev) [inline test.js:1]: Hey you forgot the ; it is good practice to have

**Review by VipinDevelops — COMMENTED** [review]
Hello thsi is review

**Review by Twitter-Dev — APPROVED** [review]

**Review by Twitter-Dev — APPROVED** [review]

**Comment by VipinDevelops** [comment]
:+1:

**Pull request merged**